src/components/templates/Search/sort.module.css

```diff
@@ -0,0 +1,31 @@
+.sortList {
+  display: inline-flex;
+  float: right;
+}
+
+.sorted {
+  display: inline-flex;
+  padding: calc(var(--spacer) / 4);
+  border: 1px solid var(--border-color);
+  text-transform: uppercase;
+  border-radius: var(--border-radius);
+  margin-right: calc(var(--spacer) / 4);
+  font-weight: var(--font-weight-bold);
+  color: var(--color-secondary);
+  background: var(--background-body);
+  cursor: pointer;
+}
+
+.selected {
+  color: var(--font-color-text);
+}
+
+.direction {
+  background: transparent;
+  border: none;
+  color: var(--font-color-text);
+  font-size: 1.3em;
```

Comment:
I used `font-size: .75rem;` to get that size seen in my screenshot. Fine making it smaller when the whole element is  a button and not just the arrow